### DISCUSSION TITLE
[BUG] Cannot override Settings ..

### DISCUSSION THREAD

**Discussion by maffi91** [rspamd/rspamd#3668] [General]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Read about bug reporting in general: https://rspamd.com/doc/faq.html#how-to-report-bugs-found-in-rspamd
    * Enabled relevant debugging logs: https://rspamd.com/doc/faq.html#how-to-debug-some-module-in-rspamd 
    * Checked the FAQs about Core files in case of fatal crash: https://rspamd.com/doc/faq.html#how-to-figure-out-why-rspamd-process-crashed
    * Tried ASAN package and obtained the ASAN report (if possible): https://rspamd.com/doc/faq.html#asan-builds 
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Arspamd
    * Checked that there is not already an experimental package or master branch

**Describe the bug**
Cannot override Settings like regex

### Steps to Reproduce

1. I create a file like this: /etc/rspamd/local.d/regexp.conf
`	MISSING_TO {
            score = 6;
    }

    SUBJECT_HAS_CURRENCY {
        score = 3;
    }`
2. systemctl reload rspamd.service
3. rspamadm configdump
4. The results are the conf add the setting, not override : 
`regexp {
    SUBJECT_HAS_CURRENCY {
        score = 3;
        score = 1;
        description = "Subject contains currency";
        re = "Subject=/[$€$¢¥₽]/Hu";
        group = "headers";
    }
...
    MISSING_TO {
        score = 6;
        score = 2;
        description = "To header is missing";
        group = "headers";
        re = "!raw_header_exists(To)";
        mime_only = true;
    }`

After that i move it to /etc/rspamd/override.d/ with the same result :( Any ideas?

### Versions

Rspamd daemon version 2.7
Debian GNU/Linux 10 (buster)

**Comment by vstakhov** [comment]
Not an issue

**Comment by vstakhov** [comment]
https://rspamd.com/doc/faq.html#how-to-change-score-for-some-symbol

**Reply by maffi91** [reply]
Thanks for the fast answer. But is it normal i see now two entries with "rspamadm configdump":

The results now:
symbols {
    MISSING_TO {
        score = 6;
    }
    SUBJECT_HAS_CURRENCY {
        score = 3;
    }
}

and litte bit later:
    ungrouped {
        symbols {
            MISSING_TO {
                score = 6;
            }
            SUBJECT_HAS_CURRENCY {
                score = 3;
            }
        }

is that normal?
